### DISCUSSION TITLE
🎉 Exciting News for the Community! Release of INP2RAD 🎉

### DISCUSSION THREAD

**Discussion by elequiniou** [OpenRadioss/OpenRadioss#3222] [Announcements]
We're thrilled to announce the release of a free converter tool that bridges the gap between .inp files (popular format supported by Abaqus and other CAE tools) and .rad files (Radioss block format), aiming to :
**Democratize further usage of explicit dynamics** by making seamless reuse of .inp format models with OpenRadioss solver, enabling cutting-edge capabilities for crash, impact, and highly nonlinear simulations.
**Unlocks exciting open-source workflows** by allowing you to prepare models with **FreeCAD** and **PrePoMax**,  2 well-known open-source CAE tools, empowering a broader community to create and analyze models efficiently.

Kudos to [Pauly](https://github.com/PaulAltair) who developed this converter:
🔗 https://github.com/OpenRadioss/tools/tree/main/input_converters/inp2rad
Try it out, share your feedback, and let us know how you're using this tool to push the boundaries of simulation!
Notice the converter can be seamlessly used from the [OpenRadioss GUI](https://github.com/OpenRadioss/Tools/tree/main/openradioss_gui)

Kudos to [Aleksander](https://www.linkedin.com/in/aleksander-sadowski-freecad/) who beta tested this tool to create this nice case study illustrating this powerful combination:
🔗 https://www.linkedin.com/posts/aleksander-sadowski-freecad_freecad-opensource-maschinenbau-activity-7272852688470511616-bUKa?utm_source=share&utm_medium=member_desktop

Kind Regards,

Eric & Marian

**Comment by SergioP1975** [comment]
That is a very good news! Could you provide some hints on how must be prepared the model in Prepomax to use the converter?

**Reply by PaulAltair** [reply]
essentially, nothing special, standard PrePoMax workflow should work, most common keywords are supported, contact pairs, ties, loads, initial conditions, constraints etc. For run type, set as 'dynamic', and the termination time and timestep values should be mapped. export as calculix (.inp) and run in OpenRadioss gui with the new inp2rad tool installed. 

Notes:
If applying CLOAD then default is to apply at the requested value from Time 0, if you want to ramp a load on over run time, specify an *AMPLITUDE that does that

There is one special workaround implemented, if you create a 'Surface Interaction' with name 'RADIOSS_GENERAL' (no need to create an actual contact) then you will get a global general contact generated in OpenRadioss (there is not a default way to create one in PrePoMax I don't think)

**Reply by SergioP1975** [reply]
Thanks!

**Reply by PaulAltair** [reply]
Having played with PrePoMax over the holidays a little, some extra tips! Don't assign 2nd order mesh to shell parts (Radioss doesn't have 2nd order shells) and when assigning properties (sections) to parts, do so 'by Part' in PrePoMax, the 'by selection' method doesn't work with the inp2rad reader, I know why, and could possibly fix, but it isn't simple and may break some other workflows, so I have for now put in a pull request to warn if this assignment method is used (Since the workaround in PrePoMax to use part assignment is relatively simple)

**Comment by PaulAltair** [comment]
The script is 'beta' but should work for many existing .inp models (created with any preprocessor for .inp format)

Known Limitations: The model must be a single file (includes not supported) and Part/Instance models are not supported ('Flat' models only)

I added a simple example model in .inp format on the confluence model pages to demo the new tool:

[Phone Drop Model (.inp format)](https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/516947969/Cell+Phone+Drop+Test+.inp+format)

**Comment by SergioP1975** [comment]
Thanks, will test as soon as possible.

**Comment by RPJ31195** [comment]
can it convert ls pre post modell .key ??

**Reply by elequiniou** [reply]
This converter is for .inp input file, for .key, you can directly run OpenRadioss with it, no need to convert.
Hope this helps.
Eric
